conftest.py

```
@@ -0,0 +1,141 @@
+@pytest.fixture
+def api_client():
+    """Return a DRF APIClient instance."""
+
+    return APIClient()
+
+
+@pytest.fixture
+def user_factory(django_user_model):
+    """Factory for creating users."""
+
+    def create_user(**kwargs):
```

Ruff (0.14.0)

Missing return type annotation for private function `create_user`

(ANN202)